catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/operation/JdbcDatabaseOperations.java

```diff
@@ -123,8 +125,12 @@ public boolean exist(String databaseName) {
 
   protected String generateDatabaseExistSql(String databaseName) {
```

Comment:
Can we add a deprecated label to this method if not for safe deleting it, since it's been replaced by `generateDatabaseExistPreparedSql()`?

Reply:
I'd keep the method name the same and alter it rather than add a new method.